apps/golden-sample-app/src/app/app.component.html

```diff
@@ -47,7 +47,6 @@
     <ul *ngIf="isAuthenticated" class="bb-layout__horizontal-nav">
       <li
         class="bb-layout__horizontal-nav-item"
-        *bbIfEntitlements="triplets.canViewTransfer"
```

Comment:
Why did you remove this `bbIfEntitlements` directive usage?

Reply:
Because entitlements for payments capability are not configured fully.
Basically there is no entitlement with such meaning as we want (Gabe is working on it)

Reply:
Please create an issue to tackle this later.